docs/application/dotnet/guides/nui/view.md

```diff
@@ -10,11 +10,13 @@
 `Position`, `Orientation`, and `Scale` of `View` are known as transforms.
 `View` supports hierarchical structure. Therefore, `Position`, `Orientation`, and `Scale` of every `View` are affected by their parents.
 
-`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and **TopLeft** position of `View`. `ParentOrigin` is a property that defines the reference point in the parent `View`. For example, if a `View` object has properties `Position` (5, 3) and `ParentOrigin` is **Center**, then the **TopLeft** of the object is located (5, 3) away from **Center** position of its parent `View` object.
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child, the default value is **TopLeft**, for instance, if the `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.
```

Comment:
@nawab-samsung, please ensure you break such sentences for readability:

The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child.  The default value is **TopLeft**.  
For example, if the `Position` property of the `View` is set to (5, 3) coordinates, `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.

OR

The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a property that defines the anchor point of the child.  The default value is **TopLeft**.  

For example, if the :
- `Position` property of the `View` is set to (5, 3) coordinates
- `ParentOrigin` property is set to **Center** 
- `PivotPoint` property is set to **TopLeft**, 
 
Then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent's `View` object.

Question: 
 The default value is **TopLeft**.  Default value of what? ParentOrigin or PivotPoint?

Reply:
@MathewP1

Request you to kindly answer  the question.

Reply:
Default is **TopLeft** for both. I've added this to guide.